modin/experimental/xgboost/test/test_xgboost.py

```diff
@@ -0,0 +1,172 @@
+# Licensed to Modin Development Team under one or more contributor license agreements.
+# See the NOTICE file distributed with this work for additional information regarding
+# copyright ownership.  The Modin Development Team licenses this file to you under the
+# Apache License, Version 2.0 (the "License"); you may not use this file except in
+# compliance with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+
+import pytest
+
+import modin.experimental.xgboost as xgb
+import modin.pandas as pd
+import numpy as np
+import xgboost
+
+from sklearn.datasets import (
+    load_iris,
+    load_diabetes,
+    load_digits,
+    load_wine,
+    load_breast_cancer,
+)
+from sklearn.metrics import accuracy_score, precision_score
+from numpy.random import uniform, randint
+
+import multiprocessing as mp
+
+
+num_cpus = mp.cpu_count()
+
+
+@pytest.mark.parametrize(
+    "num_round",
+    [2, 4, 8, 16, 32],
+    ids=["2", "4", "8", "16", "32"],
```

Comment:
3 different values will be enough. `num_round` should be more to increase accuracy of model

Reply:
Now num_round have values - [100, 500, 1000]

Reply:
I'm not sure, that these too big `num_round` significantly increases the accuracy of model for the small datasets, which we use. Could you explain, why did you choose these numbers and we can't use a set `[10, 100, 300]` e.g.? It's significantly decrease time of training.

Reply:
Now num_round=[10, 100, 300]